From: "Milan P. Gandhi" <mgandhi@redhat.com>
Date: Tue, 12 Mar 2019 18:21:11 +0530
Subject: scsi: qla2xxx: Fix comment alignment in qla_bsg.c
Patch-mainline: v5.2-rc1
Git-commit: f65c33890b95ce24f12f88c30625d678443020bb
References: bsc#1123034 bsc#1131304 bsc#1127988

Fixed a minor formatting issue with comment section in qla_bsg.c

Signed-off-by: Milan P. Gandhi <mgandhi@redhat.com>
Signed-off-by: Martin K. Petersen <martin.petersen@oracle.com>
Acked-by: Daniel Wagner <dwagner@suse.de>
---
 drivers/scsi/qla2xxx/qla_bsg.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/scsi/qla2xxx/qla_bsg.c
+++ b/drivers/scsi/qla2xxx/qla_bsg.c
@@ -1,4 +1,4 @@
-	/*
+/*
  * QLogic Fibre Channel HBA Driver
  * Copyright (c)  2003-2014 QLogic Corporation
  *
